plugins/techdocs-backend/src/helpers.ts

```diff
@@ -124,9 +125,13 @@ export const checkoutGitRepository = async (
   const user =
     process.env.GITHUB_PRIVATE_TOKEN_USER ||
     process.env.GITLAB_PRIVATE_TOKEN_USER ||
+    process.env.AZURE_PRIVATE_TOKEN_USER ||
     '';
   const token =
-    process.env.GITHUB_TOKEN || process.env.GITLAB_PRIVATE_TOKEN_USER || '';
+    process.env.GITHUB_TOKEN ||
+    process.env.GITLAB_PRIVATE_TOKEN_USER ||
```

Comment:
This seems like a bug. It should probably be `GITLAB_TOKEN` right?

Reply:
Could be, but we can merge this and fix that when we know for sure. :)